src/dndbeyond/content-scripts/character.js

```diff
@@ -35,6 +35,8 @@ function rollSkillCheck(paneClass) {
         roll_properties["reliableTalent"] = true;
     if (character.hasClassFeature("Silver Tongue"))
         roll_properties["silverTongue"] = true;
+    if (character.getSetting("indomitable-might") && ability == "STR")
+        roll_properties["indomitableMight"] = true;
```

Comment:
Why are you checking the setting? Isn't this tied to a class feature? You need to check if they have the actual feature.
Also, it's not a toggleable thing, it's always on at lvl 18, so it shouldn't have been made into a setting at all.